commit 57785c952444e4cb9d13e43d4f9cc9713d66958b
Author: Martin Pitt <martin.pitt@ubuntu.com>
Date:   Mon Jan 16 16:00:18 2012 +0100

    Use local include files for building GIR
    
    Add -I argument for g-ir-scanner to use the include files from the local tree
    instead of the ones installed in the system.

diff --git a/libxklavier/Makefile.am b/libxklavier/Makefile.am
index 6c8631a..c53ff6d 100644
--- a/libxklavier/Makefile.am
+++ b/libxklavier/Makefile.am
@@ -94,7 +94,7 @@ introspection_sources = $(xklavier_headers) $(filter %.c, $(libxklavier_la_SOURC
 
 Xkl-1.0.gir: libxklavier.la
 Xkl_1_0_gir_INCLUDES = GObject-2.0 xlib-2.0
-Xkl_1_0_gir_CFLAGS = $(INCLUDES) $(X_CFLAGS) $(XML_CFLAGS) $(GLIB_CFLAGS) $(XINPUT_CFLAGS)
+Xkl_1_0_gir_CFLAGS = -I$(top_srcdir) $(INCLUDES) $(X_CFLAGS) $(XML_CFLAGS) $(GLIB_CFLAGS) $(XINPUT_CFLAGS)
 Xkl_1_0_gir_LIBS = libxklavier.la
 Xkl_1_0_gir_FILES = $(xklavier_headers) $(introspection_sources)
 INTROSPECTION_GIRS += Xkl-1.0.gir
